paparazzi/src/test/java/app/cash/paparazzi/internal/resources/PseudolocaleGeneratorTest.kt

```diff
@@ -0,0 +1,209 @@
+import com.android.ide.common.rendering.api.ResourceNamespace
+import com.android.ide.common.rendering.api.ResourceValue
+import com.android.ide.common.resources.KnownNamespacesMap
+import com.android.ide.common.resources.ResourceValueMap
+import com.android.ide.common.resources.configuration.LocaleQualifier
+import com.android.resources.ResourceType
+import com.google.common.collect.Table
+import com.google.common.collect.Tables
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import java.util.EnumMap
+
+class PseudolocaleGeneratorTest {
+  private val resFolderRoot = resolveProjectPath("src/test/resources/folders/res")
+
+  private val repository = ResourceFolderRepository(
+    resFolderRoot.toFile(),
+    ResourceNamespace.TODO()
+  )
+
+  private fun createResourceTable(): Table<ResourceNamespace, ResourceType, ResourceValueMap> {
+    val map: Map<ResourceNamespace, Map<ResourceType, ResourceValueMap>> = KnownNamespacesMap()
+    val table = Tables.newCustomTable(map) { EnumMap(ResourceType::class.java) }
+    repository.allResources.groupBy { it.type }.forEach { (type, items) ->
+      table.put(
+        ResourceNamespace.TODO(),
+        type,
+        ResourceValueMap.create().apply {
+          putAll(items.map { it.name to it as ResourceValue })
+        }
+      )
+    }
+
+    return table
+  }
+
+  @Test
+  fun testAccentPseudolocalization() {
+    val result = createResourceTable()
+    result.pseudolocalizeIfNeeded(LocaleQualifier.getQualifier("en-rXA"))
+
+    // STRING
+    with(result[ResourceNamespace.TODO(), ResourceType.STRING]!!) {
+      assertThat(size).isEqualTo(4)
+      assertThat(get("string_name")?.value).isEqualTo("[Ţéšţ Šţŕîñĝ one two]")
+      assertThat(get("string_name_xliff")?.value).isEqualTo("[Ţéšţ Šţŕîñĝ »{0}« ŵîţĥ šûƒƒîх one two three]")
+      assertThat(get("string_name_html")?.value).isEqualTo("[Ţĥîš Ţéšţ Šţŕîñĝ one two three]")
+      assertThat(get("string_name_html")?.rawXmlValue).isEqualTo("[Ţĥîš <b>Ţéšţ</b> Šţŕîñĝ one two three]")
+      assertThat(get("string_not_translate")?.value).isEqualTo("Not Translatable String")
+    }
+
+    // ARRAY
+    with(result[ResourceNamespace.TODO(), ResourceType.ARRAY]!!) {
+      assertThat(size).isEqualTo(1)
+      with(get("string_array_name") as ArrayResourceValue) {
+        assertThat(elementCount).isEqualTo(2)
+        assertThat(getElement(0)).isEqualTo("[Fîŕšţ Ţéšţ Šţŕîñĝ one two three]")
+        assertThat(getElement(1)).isEqualTo("[Šéçöñð Ţéšţ Šţŕîñĝ one two three]")
+      }
+    }
+
+    // PLURALS
+    with(result[ResourceNamespace.TODO(), ResourceType.PLURALS]!!) {
+      assertThat(size).isEqualTo(1)
+      with(get("plural_name") as PluralsResourceValue) {
+        assertThat(pluralsCount).isEqualTo(2)
+        assertThat(getValue(0)).isEqualTo("[Ñöţĥîñĝ one two]")
+        assertThat(getValue(1)).isEqualTo("[Öñé Šţŕîñĝ one two]")
+      }
+    }
+
+    // Other resources should not be pseudo localized
```

Comment:
nit: should the below resource types be moved out into their own test method?  i could go either way!

something like:
```
@Test
fun doesNotPseudolocalizeOtherResourceTypes() {
    val table = createResourceTable()

    table.pseudolocalizeIfNeeded(LocaleQualifier.getQualifier("en-rXA"))
    // BOOL
    // COLOR
    // DIMEN

    table.pseudolocalizeIfNeeded(LocaleQualifier.getQualifier("ar-rXB"))
    // BOOL
    // COLOR
    // DIMEN
}